lib/zwave/ZwaveDevice.js

```diff
@@ -701,32 +782,41 @@ class ZwaveDevice extends MeshDevice {
 		this._capabilities[capabilityId][commandClassId] = this._capabilities[capabilityId][commandClassId] || {};
 
 		// merge systemOpts & userOpts
-		let systemOpts = {};
-		try {
+		this._mergeSystemAndUserOpts(capabilityId, commandClassId, userOpts);
 
-			// First try get device class specific system capability
-			systemOpts = this._getDeviceClassSpecificSystemCapability(capabilityId, commandClassId);
+		// register get/set/realtime
+		this._registerCapabilityRealtime(capabilityId, commandClassId);
+		this._registerCapabilitySet(capabilityId, commandClassId);
+		this._registerCapabilityGet(capabilityId, commandClassId);
+	}
 
-			// If not available use general system capability
-			if (!systemOpts) systemOpts = require(`./system/capabilities/${capabilityId}/${commandClassId}.js`);
+	registerMultipleCapabilities(capabilitiesOpts = [], fn) {
 
-		} catch (err) {
-			if (err.code !== 'MODULE_NOT_FOUND') {
-				process.nextTick(() => {
-					throw err;
-				});
+		// Loop all provided capabilities
+		capabilitiesOpts.forEach(capabilityObj => {
+			const capabilityId = capabilityObj.capability;
+			const commandClassId = capabilityObj.commandClass;
+			const userOpts = capabilityObj.opts || {};
+
+			// Check if device has the command class we're trying to register, if not, abort
+			if (typeof this.node.CommandClass[`COMMAND_CLASS_${commandClassId}`] === 'undefined') {
+				return this.error('Invalid commandClass:', commandClassId);
 			}
-		}
 
-		this._capabilities[capabilityId][commandClassId] = Object.assign(
-			systemOpts || {},
-			userOpts || {}
-		);
+			// register the Z-Wave capability listener
+			this._capabilities[capabilityId] = this._capabilities[capabilityId] || {};
+			this._capabilities[capabilityId][commandClassId] = this._capabilities[capabilityId][commandClassId] || {};
 
-		// register get/set/realtime
-		this._registerCapabilityRealtime(capabilityId, commandClassId);
-		this._registerCapabilitySet(capabilityId, commandClassId);
-		this._registerCapabilityGet(capabilityId, commandClassId);
+			// merge systemOpts & userOpts
+			this._mergeSystemAndUserOpts(capabilityId, commandClassId, userOpts);
+
+			// register get/set/realtime
+			this._registerCapabilityRealtime(capabilityId, commandClassId);
+			this._registerCapabilityGet(capabilityId, commandClassId);
+
+			// Register debounced capabilities set
+			this._registerCapabilitiesSet(capabilitiesOpts, fn);
```

Comment:
I don't think this is supposed to be in the loop? Since it gets `capabilitiesOpts` as parameter, not `capabilityOpt`, it would register all capabilities multiple times right?